community/ndp/kernelextension/src/test/java/org/neo4j/ext/MjolnirExtensionIT.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.ext;
+
+import org.junit.After;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.net.ConnectException;
+import java.net.URI;
+
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.factory.GraphDatabaseFactory;
+
+public class MjolnirExtensionIT
+{
+    @Rule
+    public TemporaryFolder tmpDir = new TemporaryFolder();
+    private GraphDatabaseService db;
+
+    @Test
+    public void shouldLaunchMjolnir() throws Throwable
+    {
+        // When I run Neo4j with the mjolnir extension on the class path, and experimental mjolnir config on
+        db = new GraphDatabaseFactory()
+                .newEmbeddedDatabaseBuilder( tmpDir.getRoot().getAbsolutePath() )
+                .setConfig( NDPKernelExtension.Settings.mjolnir_enabled, "true" )
+                .newGraphDatabase();
```

Comment:
Can't you use `TestGraphDatabaseFactory`? It has more appropriate default settings for testing.